mmcv/cnn/bricks/conv2d_samepadding.py

```diff
@@ -0,0 +1,61 @@
+import math
+
+from torch import nn
+from torch.nn import functional as F
+
+from .registry import CONV_LAYERS
+
+
+@CONV_LAYERS.register_module()
+class Conv2dSamePadding(nn.Conv2d):
```

Comment:
Users may be confused with the class name. It sounds like Conv2d with "replicate" padding mode.

Reply:
How about using `Conv2dAdaptivePadding`? Because it will pad the image width (height) with zeros up to a multiple of the filter width (height) when the image width is not a multiple of the filter width or the image height is not a multiple of the filter height in an adaptive way.

Reply:
Sounds good.

Reply:
The class mane has been changed to `Conv2dAdaptivePadding` in the latest version. Please have a check.